# This service requests the RoadmapGenerator to create a new roadmap graph.

WarehouseConfiguration warehouse_configuration
---
bool success
